### DISCUSSION TITLE
Permanant vs temporary registrations

### DISCUSSION THREAD

**Discussion by iaindillingham** [opensafely/documentation#716] [Q&A] [answered]
Does the [OpenSAFELY-TPP database][1] include only patients who are permanently registered at a practice? Or does it also include patients who are temporarily registered at a practice?

The [_Characteristics of patients included in the OpenSAFELY-TPP database_][2] notebook includes a section entitled _registered patients_. This counts "actively registered" patients, but it's unclear how "actively registered" is defined.

[1]: https://docs.opensafely.org/dataset-systmone/
[2]: https://github.com/opensafely/database-notebooks/blob/master/notebooks/database-patient-characteristics.ipynb

**Comment by wjchulme** [comment, marked as answer]
Regarding the content of the notebook, "actively registered" might just as well be expressed by "registered"! The count is derived from the [`registered_with_one_practice_between`](https://docs.opensafely.org/study-def-variables/#cohortextractor.patients.registered_with_one_practice_between) function, and you can see the study definition code [here](https://github.com/opensafely/database-notebooks/blob/c214ad721db7b693416b6d908fcc9eababf426f0/analysis/study_definition.py#L34). 

But then what is "registered"? [This thread](https://bennettoxford.slack.com/archives/C010SJ89SA3/p1603188749464600) suggests only "full GMS" registrations are included in the OpenSAFELY-TPP database -- so no temporary or other registration types. I assume GMS refers to the [GMS1 form](https://www.gov.uk/government/publications/gms1) used to register with a practice. So the count in the notebook is presumably for "permanent" (or "full GMS") registrations. The thread is two years old so things may have changed since.
